examples/fabric/tensor_parallel/data.py

```diff
@@ -0,0 +1,19 @@
+import torch
+from torch.utils.data import Dataset
+
+
+class RandomTokenDataset(Dataset):
+    def __init__(self, vocab_size: int, seq_length: int):
+        self.vocab_size = vocab_size
+        self.seq_length = seq_length
+        self.tokens = torch.randint(
+            self.vocab_size,
+            size=(len(self), self.seq_length + 1),
+            generator=torch.Generator().manual_seed(42),
```

Comment:
Can we add a comment or info somewhere about why seeding is important to have and how it plays with tp + dp?